cmake/Config.cmake

```diff
@@ -19,8 +19,24 @@ if (WIN32)
             "${CMAKE_CURRENT_SOURCE_DIR}/cs/test/compat/bin/retail")
 endif()
 
-find_package (PythonLibs 2.7)
-find_package (PythonInterp 2.7)
+# find python interpreter and library
+# to specify a different version, invoke cmake with:
+# -DPython_ADDITIONAL_VERSIONS=Major.Minor
+find_package (PythonInterp ${Python_ADDITIONAL_VERSIONS})
```

Comment:
I think you don't need to/shouldn't specify `${Python_ADDITIONAL_VERSIONS}` explicitly. It is added by `find_package` to the front of the list of versions to consider. I *think* we might still need to specify 2.7 as minimum version though.